tests/api/runtimes/test_base.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+
+import mlrun.errors
+from mlrun.runtimes.base import BaseRuntime
+from tests.api.runtimes.base import TestRuntimeBase
+
+
+class TestBaseRunTime(TestRuntimeBase):
+    def custom_setup_after_fixtures(self):
+        self._mock_create_namespaced_pod()
+
+    @pytest.mark.parametrize(
+        "inputs", [{"input1": 123}, {"input1": None}, {"input1": None, "input2": 2}]
+    )
+    def test_run_with_invalid_inputs(self, db: Session, client: TestClient, inputs):
+        runtime = BaseRuntime()
+        with pytest.raises(mlrun.errors.MLRunInvalidArgumentTypeError):
+            self._execute_run(runtime, inputs=inputs)
+
+    def test_run_with_currect_inputs(self, db: Session, client: TestClient):
```

Comment:
First of all you have a typo it's correct not currect, but anyways if the above test tests the invalid scenario, I would go here with the opposite word - valid - so `test_run_with_valid_inputs`